### ISSUE TITLE
[Accessibility] Cannot merge PR via keybinding

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode-pull-request-github#5765]
Type: <b>Bug</b>

## Reproducible Steps

1. Add `  "githubPullRequests.defaultMergeMethod": "squash"` to `settings.json`.

1. Assign a keybinding to `pr.merge` command like below:

```json
[
  {
    "key": "ctrl+shift+g ctrl+shift+m",
    "command": "pr.merge",
    "when": "!isMac"
  },
  {
    "key": "cmd+shift+g cmd+shift+m",
    "command": "pr.merge",
    "when": "isMac"
  }
]
```

1. Checkout to any PR to merge.

1. Press the assigned keybinding.

* Note: The squash merge does not work.

Extension version: 0.81.2024021509
VS Code version: Code - Insiders 1.87.0-insider (d84c946d27b649d614319c796911871320befd42, 2024-02-16T06:04:29.948Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (4.26GB free)|
|Process Argv|--crash-reporter-id b05b88e5-8894-4031-ae34-fa034ebddea9|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vscod805cf:30301675
vsaa593:30376534
py29gd2263:30784851
c4g48928:30535728
a9j8j154:30646983
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
h48ei257:30962250
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
pydisww2:30959801
cppperfcontrol:30963551
d34g3935:30961436
edj9j583:30943796
fegfb526:30952798
7j2b6412:30951517
e3gdj431:30946825
ccp2r3:30958153
pythonaic:30966029

```

</details>

**Comment by alexr00** [comment]
duplicate ofhttps://github.com/microsoft/vscode-pull-request-github/issues/5527
